### PR TITLE
RELATED: WEB-1719, RAIL-2778 create version 8.1.0

### PR DESCRIPTION
**Pull request by no23reason**
Also, Live Examples were changed to Examples Gallery in the sidebar according to WEB-1719.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/567/ : SUCCESS in 1m 38s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/340/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/568/ : SUCCESS in 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/341/ : SUCCESS in 5s

File: website/versioned_docs/version-8.1.0/10_vis__theme_provider.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+sidebar_label: Theme Provider
+copyright: (C) 2007-2020 GoodData Corporation
+id: version-8.1.0-theme_provider
+original_id: theme_provider
+---
+
+The **Theme Provider component** allows you to customize the visual style of your dashboards by applying a theme.
+
+To be able to use the Theme Provider component, set up the `backend` and `workspace` properties in your application (see [Connecting to an Analytical Backend](02_start__connecting_backend.md#initial-setup)).
+
```
Comment (lupko) [inline website/versioned_docs/version-8.1.0/10_vis__theme_provider.md:12]: I don't think this is true. The following text contradicts it as well.

File: website/versioned_docs/version-8.1.0/10_vis__theme_provider.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+<ThemeProvider backend={backend} workspace="your-project-id">
+    <Application {...}>
+</ThemeProvider>
+```
+
+### Using the Theme Provider component with a custom theme
+
+You can create properties of a custom theme, and pass the theme to the `ThemeProvider` that you added to your application.
+
```
Comment (lupko) [inline website/versioned_docs/version-8.1.0/10_vis__theme_provider.md:30]: we could use a link to repository ITheme definition which has full description of the props

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/570/ : SUCCESS in 1m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/343/ : SUCCESS in 6s

File: docs/10_vis__theme_provider.md [hunk 3]
```diff
@@ -13,7 +13,24 @@ To be able to use the Theme Provider component, set up the `backend` and `worksp
...
+        <ThemeProvider>
+            <Application {...}>
+        </ThemeProvider>
+    </WorkspaceProvider>
+</BackendProvider>
+```
+
+### Using the Theme Provider component without backend and workspace providers
+
+When using the Theme Provider component without backend and workspace providers, you also need to pass the `backend` and `workspace` props to Theme Provider.
```
Comment (evgeniabogranskaya) [inline docs/10_vis__theme_provider.md:33]: When using the Theme Provider component without backend and workspace providers, you have to pass the `backend` and `workspace` props to Theme Provider.

File: docs/10_vis__theme_provider.md [hunk 4]
```diff
@@ -25,7 +42,7 @@ import { ThemeProvider } from "@gooddata/sdk-ui-theme-provider";
 
 ### Using the Theme Provider component with a custom theme
 
-You can create properties of a custom theme, and pass the theme to the `ThemeProvider` that you added to your application.
+You can create properties of a custom theme, and pass the theme to the `ThemeProvider` that you added to your application (see [the declaration of ITheme](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-backend-spi/src/workspace/styling/theme.ts#L178) to see all the available properties).
```
Comment (evgeniabogranskaya) [inline docs/10_vis__theme_provider.md:45]: You can create properties of a custom theme, and pass the theme to the `ThemeProvider` that you added to your application (for all available properties, see [the declaration of ITheme](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-backend-spi/src/workspace/styling/theme.ts#L178)).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/571/ : SUCCESS in 1m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/344/ : SUCCESS in 6s

**Review by lupko — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/572/ : SUCCESS in 1m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/70/ : SUCCESS in 2m 29s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/163/ : SUCCESS in 19s

**Pull request merged**